index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lisandro-Subrat</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="container">
+      <div id="header">
+        <h1 id="title">survey</h1>
+        <p id="description">Thanks for filling the survey!</p>
+      </div>
+
+      <form id="survey-form">
+        <div class="form-control">
+          <label id="name-label">Name:</label>
```

Comment:
You made it so well using labels, but is a good practice to add the 'for' attribute to the labels, and its going to make the HTML more understandable for the search engine :blush: , you should try it like this:
```suggestion
          <label for='name' id="name-label">Name:</label>
          <input
            type="text"
            placeholder="Enter your name"
            name="name"
            id="name"
            required
          />
```